test/dataframe.jl

```diff
@@ -325,25 +325,28 @@ module TestDataFrame
     df = DataFrame(a=Union{Int, Missing}[1, 2, 3], b=Union{Float64, Missing}[3.0, 4.0, 5.0])
     @test deleterows!(df, [2, 3]) === df
     @test df == DataFrame(a=[1], b=[3.0])
-
-    # describe
-    #suppress output and test that describe() does not throw
-    devmissing = Compat.Sys.isunix() ? "/dev/null" : "nul"
-    open(devmissing, "w") do f
-        @test nothing == describe(f, DataFrame(a=[1, 2], b=Any["3", missing]))
-        @test nothing ==
-              describe(f, DataFrame(a=Union{Int, Missing}[1, 2],
-                                    b=["3", missing]))
-        @test nothing ==
-              describe(f, DataFrame(a=CategoricalArray([1, 2]),
-                                    b=CategoricalArray(["3", missing])))
-        @test nothing == describe(f, [1, 2, 3])
-        @test nothing == describe(f, [1, 2, 3])
-        @test nothing == describe(f, CategoricalArray([1, 2, 3]))
-        @test nothing == describe(f, Any["1", "2", missing])
-        @test nothing == describe(f, ["1", "2", missing])
-        @test nothing == describe(f, CategoricalArray(["1", "2", missing]))
-    end
+    
+    # Test describe
+    struct MySuperLongNameForAStructIsThisToolong end
+    @testset "describe(df)" begin
+        describe_output = DataFrame(variable = [:number, :number_missing, :non_number, :non_number_missing, :long_struct],
+                                    mean = [2.5, 2.0, nothing, nothing, nothing],
+                                    min = [1.0, 1.0, nothing, nothing, nothing],
+                                    median = [2.5, 2.0, nothing, nothing, nothing],
+                                    max = [4.0, 3.0, nothing, nothing, nothing],
+                                    nmissing = [nothing, 1, nothing, 1, nothing],
+                                    eltype= [Int64, Int64, String, String, MySuperLongNameForAStructIsThisToolong])
```

Comment:
Add space before `=`.

Reply:
Ah, and use `Int` rather than `Int64` (this explains the 32-bit Windows failure).